src/Avalonia.Visuals/Rendering/SceneGraph/DrawOperation.cs

```diff
@@ -11,7 +11,10 @@ internal abstract class DrawOperation : IDrawOperation
     {
         public DrawOperation(Rect bounds, Matrix transform)
         {
+            bounds = bounds.Normalize();
+
             bounds = bounds.TransformToAABB(transform);
+
```

Comment:
Can combine this lines:

```suggestion
            bounds = bounds.Normalize().TransformToAABB(transform);
```

Reply:
done